app/lib/bing_api.rb

```diff
@@ -0,0 +1,54 @@
+require 'net/https'
+require 'uri'
+require 'json'
+require 'pry'
+
+class BingApiV7
```

Comment:
you have incorrect name for this file location
if you have file named `bing_api.rb` so your class must be `BingApi`
read more about [Rails auto-loading constants](http://guides.rubyonrails.org/autoloading_and_reloading_constants.html)